src/components/InfoTip/InfoTip.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from '../Dropdown';
+
+class InfoTip extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnKeyDown = this.handleOnKeyDown.bind(this);
+    this.handleOnClick = this.handleOnClick.bind(this);
+    this.state = { open: false, footer_focused: false };
+  }
+
+  handleOnKeyDown(event) {
+    switch (event.keyCode) {
+      case 13:
```

Comment:
using 13 and 9 make it hard to understand.

#190 added those variables into a helper file:
https://github.com/patternfly/patternfly-react/pull/190/files#diff-6d029458f6215790b0dcdb4b6585f3c5R40

I think it will also make it more reasonable to understand if you will split the different cases into different functions as well:
```jsx
switch (event.keyCode) {
  case KEY_CODES.ENTER_KEY:
    return this.handleEnterKeyPress(event);
  case KEY_CODES.TAB_KEY:
    return this.handleTabKeyPress(event);
}
```

Reply:
Great, I was looking about to add the library keycode like react-boostrap. I didn't know about this constants.

Reply:
@aljesusg they were not merged yet, so you couldn't know :)
You will need to copy/paste the code from #190 in order to have it.

Reply:
There is not a helper with KEY_CODES now. It seems it was a old PR. Anyway
```js
handleKeyDown(event) {
    switch (event.key) {
      case 'Enter':
        return this.handleEnterKeyDown(event);
      case 'Tab':
        return this.handleTabKeyDown(event);
      default:
        return
    }
  }
```